model_zoo/deepfm_functional_api/deepfm_functional_api.py

```diff
@@ -110,16 +110,13 @@ def _parse_data(record):
     return dataset
 
 
-def eval_metrics_fn(predictions, labels):
-    labels = tf.reshape(labels, [-1])
-    predictions = tf.reshape(predictions, [-1])
-    global AUC_metric
```

Comment:
Is it safe to remove "global" now?

Reply:
Yes, because `eval_metrics_fn` returns the AUC metric instance now. Master saves this instance in `_EvaluationJob`.